CMakeLists.txt

```diff
@@ -730,15 +712,9 @@ CONFIGURE_FILE(
 
 
 # Check if we want to precompile header files. This speeds up compile time,
-# but can fail on some machines with old cmake, therefore it is disabled by
-# default. Also cotire currently has issues with cmake generator expressions
-# used in deal.II for cmake version 3.9 and above. Disable cotire for cmake >=
-# 3.9. Starting with cmake 3.16 there is native support inside cmake and we can
-# use that.
-IF(CMAKE_VERSION VERSION_LESS 3.9)
-  SET(ASPECT_PRECOMPILE_HEADERS OFF CACHE BOOL "Precompile external header files using cotire to speedup compile time. Currently only supported for cmake 3.8 and older versions, or 3.16 and newer versions.")
-  SET(ASPECT_UNITY_BUILD OFF CACHE BOOL "Combine source files into less compile targets to speedup compile time. Currently only supported for cmake 3.16 and newer versions." FORCE)
-ELSEIF(NOT CMAKE_VERSION VERSION_LESS 3.16)
+# but can fail on some machines with old cmake. Starting with cmake 3.16
+# there is native support inside cmake and we can precompile headers.
+IF(NOT CMAKE_VERSION VERSION_LESS 3.16)
```

Comment:
I think it would be easier to read as a *positive* expression: `IF (CMAKE_VERSION VERSION_GREATER_EQUAL THAN 3.16)`.

Reply:
can you also update the comment "for cmake 3.8 and older" below?